code/java/support.java

```diff
@@ -0,0 +1,24 @@
+/* Support code for Java examples */
+
+package code.java;
+import java.util.ArrayList;
+
+public class support {
+
+	// Returns an ArrayList of the parsed CSV line
+	public static ArrayList<String> parseCsvLine(String csvLine) {
```

Comment:
This method doesn't replicate the c++ example, I wasn't quite sure how to design the methods in a simple manner to act similar to the c++ support functions.